client/app/account/hooks/useUpdateAccount.ts

```diff
@@ -17,16 +17,20 @@ export const useAccountEdit = (user: UserType) => {
     defaultValues: {
       name: user.name,
       email: user.email,
-      icon: user.icon,
+      icon: user.icon.toString(),
     },
   })
 
-  const { mutate, isPending } = useAccountMutation()
+  const { mutate, isPending } = useUpdateAccountMutation()
 
   const onEditAccount = (data: AccountSchema) => {
     mutate(data, {
       onSuccess: () => {
         queryClient.invalidateQueries({ queryKey: ['me'] })
+        toast({
+          variant: 'success',
+          title: 'Successfully edited your account!',
```

Comment:
Even though the page title is edit, I think here `Successfully updated your account!`  makes more sense. What do you think?

Reply:
Changed it. 09034d33d5092f700fffd23a4afd4373da786ed1